### DISCUSSION TITLE
Performance tuning of Nominatim

### DISCUSSION THREAD

**Discussion by kerchner** [osm-search/Nominatim#2418] [Q&A]
Hello,

Is there any documentation providing guidance on Nominatim server sizing - number of cores, etc.?  Is Nominatim designed so that it takes advantage of parallel processing and multiple cores?  I'm looking for ways to speed up the geocoding API.

Thanks!

**Comment by mtmail** [comment]
For the API Nominatim acts like other PHP web apps, each process will only deal with one incoming request. The usual scaling is to add a PHP accelerators like php-fpm. Then you can max out all CPU cores. See https://github.com/osm-search/Nominatim/issues/1290
